pkg/tags/tags.go

```diff
@@ -311,25 +341,47 @@ func ParseObjectXML(reader io.Reader) (*Tags, error) {
 	return unmarshalXML(reader, true)
 }
 
+func (tags *tagSet) parseTags(tgs string) (err error) {
+	for tgs != "" {
+		var key string
+		key, tgs, _ = strings.Cut(tgs, "&")
```

Comment:
Yeah, a bit too soon. When 1.20 is out we can use it :)

Reply:
done